### PR TITLE
Bump metrics-server to v0.6.1

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
tested :
Screenshot from 2022-03-15 19-07-30

File: deploy/addons/metrics-server/metrics-server-deployment.yaml.tmpl [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
         resources:
           requests:
             cpu: 100m
-            memory: 300Mi
+            memory: 200Mi
```
Comment (ArangoGutierrez) [inline deploy/addons/metrics-server/metrics-server-deployment.yaml.tmpl:43]: As recommended at 
https://github.com/kubernetes-sigs/metrics-server/blob/57c66e192791e5888a3fb27cdd9b1ab189f54908/manifests/base/deployment.yaml#L29-L32

**Comment by ArangoGutierrez** [comment]
Weird, my minikube setup is kvm + crio 
`KVM_Linux_crio — Jenkins: completed with 143 / 230 failures in 8.01 minutes.`

**Comment by spowelljr** [comment]
Could be a problem with the machine itself, we'll look into it

**Comment by ArangoGutierrez** [comment]
> Could be a problem with the machine itself, we'll look into it

Thanks, let me know how can I help.

**Comment by ArangoGutierrez** [comment]
Just checking if there's something I can do to move this PR forward :)

**Comment by ArangoGutierrez** [comment]
👋🏽

**Comment by spowelljr** [comment]
I resolved the issue with the machine, running tests again

**Pull request merged**